packages/shared/src/components/squads/SquadHeaderMenu.tsx

```diff
@@ -51,31 +59,64 @@ export default function SquadHeaderMenu({
     }
   };
 
+  const onDeleteSquad = async () => {
+    const options: PromptOptions = {
+      title: `Delete ${squad.name}`,
+      description: `Deleting ${squad.name} means you and all squad members will lose access to all posts that were shared in the Squad. Are you sure?`,
+      okButton: {
+        title: 'Delete',
+        className: 'btn-secondary',
```

Comment:
I think there is a mistake in the classnames

Reply:
@el I think it should be right?

See design:
![Screenshot 2023-01-16 at 11 32 45](https://user-images.githubusercontent.com/554874/212644994-c676b0aa-8b30-46ce-b9ac-99ca75311731.png)

Reply:
Ohh ok. I assumed Delete would have the red color or something.